subgraph/.gitignore

```diff
@@ -1,2 +1,4 @@
 generated
-schema.graphql
```

Comment:
`schema.graphql` is not tracked because its procedurally generated. 

If you want to track it, then you should include a CI test to verify is actually in sync with the config

Reply:
The idea is that all changes should go in src/schema.gql. If you start changing ./schema.gql, these changes will be overriden